lib/node_modules/@stdlib/stats/base/dists/hypergeometric/examples/index.js

```diff
@@ -19,6 +19,79 @@
 'use strict';
 
 var objectKeys = require( '@stdlib/utils/keys' );
-var hypergeometric = require( './../lib' );
+var hypergeometric = require( '@stdlib/stats/base/dists/hypergeometric' );
```

Comment:
Should still be a relative require:

```suggestion
var hypergeometric = require( './../lib' );
```